doc-content/enterprise-only/installation/clustering-bc-testing-proc.adoc

```diff
@@ -0,0 +1,26 @@
+[id='clustering-bc-testing-proc']
+= Verifying the {PRODUCT} cluster
+
+After you have configured the cluster for {PRODUCT}, create an asset to verify that the installation is working.
+
+.Procedure
+. In a web browser, enter `<node-IP-address>:8080/{URL_COMPONENT_CENTRAL}`. Replace `<node-IP-address>` with the IP address of a particular node.
+. Enter the `admin` user credentials that you created during installation. The {CENTRAL} home page appears.
+//. Select *Menu* -> *Deploy* -> *Execution Servers*.
```

Comment:
"temporarily removed" (commented out) lines should probably have a comment re why they are temporarily removed. Otherwise, if someone else ever edits the file, they will be confused.